website/students/StudentsList.js

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import students from './students';
+
+const SIZE = 128;
+
+export default function StudentsList({year}) {
+  const [hideImage, setHideImage] = useState(false);
+
+  return (
+    <div>
+      <img
+        alt="Class photo"
+        className="margin-bottom--lg"
+        onError={() => {
+          setHideImage(true);
+        }}
+        src={useBaseUrl(`/img/class-photos/${year}.jpg`)}
+        style={{
+          display: hideImage ? 'none' : 'inherit',
+        }}
+      />
+      <div className="row">
+        {students
+          .filter((batch) => batch.year === year)[0]
+          .students.map(({name, id, faculty, blog, one_liner}) => (
+            <div className="col col--4 margin-bottom--lg" key={name}>
+              <div
+                className="card shadow--tl padding-top--lg text--center"
```

Comment:
Actually, I'm thinking about the shadow - I personally feel that the drop shadow here is slightly incongruent with how the rest of the website feels, though I do understand that it's one way of showing the border of the cards. Would like to hear your thoughts on this.

Reply:
I agree. I'll change to a 1px grayish border